src/System.Runtime/tests/System/EnumTests.cs

```diff
@@ -213,18 +253,79 @@ public static void GetName_NonIntegralTypes(Type enumType, object value, string
+            yield return new object[] { typeof(Int16Enum), (short)99, false };
+
+            // UInt16
+            yield return new object[] { typeof(UInt16Enum), "One", true };
+            yield return new object[] { typeof(UInt16Enum), "None", false };
+            yield return new object[] { typeof(UInt16Enum), UInt16Enum.One, true };
+            yield return new object[] { typeof(UInt16Enum), (UInt16Enum)99, false };
+            yield return new object[] { typeof(UInt16Enum), (ushort)1, true };
+            yield return new object[] { typeof(UInt16Enum), (ushort)99, false };
+
+            // Int32
+            yield return new object[] { typeof(SimpleEnum), "Red", true };
+            yield return new object[] { typeof(SimpleEnum), "Green", true };
+            yield return new object[] { typeof(SimpleEnum), "Blue", true };
+            yield return new object[] { typeof(SimpleEnum), " Blue ", false };
+            yield return new object[] { typeof(SimpleEnum), " blue ", false };
+            yield return new object[] { typeof(SimpleEnum), SimpleEnum.Red, true };
+            yield return new object[] { typeof(SimpleEnum), (SimpleEnum)99, false };
+            yield return new object[] { typeof(SimpleEnum), 1, true };
+            yield return new object[] { typeof(SimpleEnum), 99, false };
+            yield return new object[] { typeof(Int32Enum), 0x1 | 0x02, false };
+
+            // UInt32
+            yield return new object[] { typeof(UInt32Enum), "One", true };
+            yield return new object[] { typeof(UInt32Enum), "None", false };
+            yield return new object[] { typeof(UInt32Enum), UInt32Enum.One, true };
+            yield return new object[] { typeof(UInt32Enum), (UInt32Enum)99, false };
+            yield return new object[] { typeof(UInt32Enum), (uint)1, true };
+            yield return new object[] { typeof(UInt32Enum), (uint)99, false };
+
+            // Int64
+            yield return new object[] { typeof(Int64Enum), "One", true };
+            yield return new object[] { typeof(Int64Enum), "None", false };
+            yield return new object[] { typeof(Int64Enum), Int64Enum.One, true };
+            yield return new object[] { typeof(Int64Enum), (Int64Enum)99, false };
+            yield return new object[] { typeof(Int64Enum), (long)1, true };
+            yield return new object[] { typeof(Int64Enum), (long)99, false };
+
+            // UInt64
+            yield return new object[] { typeof(UInt64Enum), "One", true };
+            yield return new object[] { typeof(UInt64Enum), "None", false };
+            yield return new object[] { typeof(UInt64Enum), UInt64Enum.One, true };
+            yield return new object[] { typeof(UInt64Enum), (UInt64Enum)99, false };
+            yield return new object[] { typeof(UInt64Enum), (ulong)1, true };
+            yield return new object[] { typeof(UInt64Enum), (ulong)99, false };
+        }
```

Comment:
And you did these as MemberData instead of InlineData just for consistency?  Or are there non-constant values here I'm just not seeing?